integration/feature/docannotations/test/src/DocAnnotationsTests.scala

```diff
@@ -3,28 +3,102 @@ package mill.integration
 import utest._
 
 object DocAnnotationsTests extends IntegrationTestSuite {
+  def globMatches(glob: String, input: String) = {
+    StringContext.glob(glob.stripMargin.split("\\.\\.\\."), input).isDefined
+  }
   val tests = Tests {
     initWorkspace()
     test("test") - {
       val res = eval("inspect", "core.test.ivyDeps")
       assert(res == true)
+
       val inheritedIvyDeps = ujson.read(meta("inspect"))("value").str
       assert(
-        inheritedIvyDeps.contains("core.test.ivyDeps"),
-        inheritedIvyDeps.contains("Overriden ivyDeps Docs!!!"),
-        inheritedIvyDeps.contains("Any ivy dependencies you want to add to this Module")
+        globMatches(
+          """core.test.ivyDeps(build.sc:...)
```

Comment:
These tests are a bit fragile, but it's probably worth having at some _some_ test where we assert on the overall shape of the inspect output. These docs and task inputs don't change very often anyway, so I expect the maintenance burden of keeping the tests up to date should be tolerable